modules/hooks.nix

```diff
@@ -87,6 +87,16 @@ in
             }";
           files = "\\.l?hs$";
         };
+      fourmolu =
+        {
+          name = "fourmolu";
+          description = "Haskell code prettifier.";
+          entry =
+            "${tools.fourmolu}/bin/fourmolu --mode inplace ${
+            lib.escapeShellArgs (lib.concatMap (ext: [ "--ghc-opt" "-X${ext}" ]) settings.ormolu.defaultExtensions)
```

Comment:
Shouldn't settings come from `fourmolu` as well?

Reply:
Well, we track upstream very closely. The CLI is unlikely to ever diverge.

We could add a `settings.fourmolu` instead with the exact same implementation, but I think I prefer it this way.